app/dashboard/templates/profile_details.html

```diff
@@ -80,12 +80,9 @@ <h2 class="mt-3">
               {% endif %}
               <div>
                 <div>
-                  <h4>{% blocktrans %}{{profile.handle}} has received these tips{% endblocktrans %}</h4>
-                  <div class="text-center">
-                    {% if tips|length == 0 %} 
-                      {% trans "No tips found" %}
-                    {% endif %} 
-                    {% if tips|length > 0 %}
+                  {% if tips|length > 0 %}
+                    <h4>{% blocktrans %}{{profile.handle}} has received these tips{% endblocktrans %}</h4>
```

Comment:
I'm not sure what `profile.handle` returns. In the screenshot I posted, you can find there wasn't anything displayed before 'has received...'  which makes me think it returns `None`. 

Why aren't we able to get the github handle?

/cc @mbeacom